felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -825,16 +871,16 @@ func (m *bpfEndpointManager) updateWEPsInDataplane() {
 	})
 }
 
-// applyPolicy actually applies the policy to the given workload.
-func (m *bpfEndpointManager) applyPolicy(ifaceName string) error {
+func (m *bpfEndpointManager) doApplyPolicy(ifaceName string, isReady *bool) error {
```

Comment:
Could you just return `(bool, error)` instead of an out param?

Reply:
I felt like this made simpler code without that much restructuring.